crates/astria-composer/tests/blackbox/helper/mock_sequencer.rs

```diff
@@ -18,6 +25,12 @@ use wiremock::{
 pub async fn start() -> MockServer {
     let server = MockServer::start().await;
     mount_abci_info_mock(&server).await;
+    mount_abci_query_mock(
```

Comment:
Do you expect this  query to be called more than once? Can you add a `.expect(1)` to it?